contracts/monitoring/src/state.rs

```diff
@@ -10,6 +11,10 @@ pub struct Config {
 
 pub const CONFIG: Item<Config> = Item::new("config");
 
-type ProverAddresses = Vec<Addr>;
+type ProverAddress = Addr;
```

Comment:
I don't think this is really necessary. Up to you, but I would just use Addr directly.